app/routes/profile.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
```

Comment:
Mix `authenticated-route-mixin` with this Route declaration so that, this page is accessible only if a user is logged in. See [`app/routes/create.js`](https://github.com/fossasia/open-event-frontend/blob/development/app/routes/create.js) for reference.